src/components/TransactionsTableCondensed.vue

```diff
@@ -50,16 +65,22 @@
 </template>
 
 <script setup>
+import { contractsHints } from '@/utils/hints/contractsHints'
 import TransactionCell from '@/components/TransactionCell'
 import ValueHashEllipsed from '@/components/ValueHashEllipsed'
 import DatetimeLabel from '@/components/DatetimeLabel'
+import { transactionsHints } from '~/utils/hints/transactionsHints'
 
 defineProps({
   transactions: {
     type: Object,
     required: true,
   },
 })
+
+function getHintKey(type) {
+  return type.charAt(0).toLowerCase() + type.slice(1)
```

Comment:
Considering it will grow by a few additional lines due to my previous suggestion I recommend to extract it to a separate file to avoid duplication

Reply:
Right, I moved it to adapters